runtime/gc_modron_standard/ContinuationObjectBufferStandard.cpp

```diff
@@ -102,3 +105,38 @@ MM_ContinuationObjectBufferStandard::reinitializeForRestore(MM_EnvironmentBase *
 	return true;
 }
 #endif /* defined(J9VM_OPT_CRIU_SUPPORT) */
+
+void
+MM_ContinuationObjectBufferStandard::iterateAllContinuationObjects(MM_EnvironmentBase *env)
+{
+	MM_GCExtensions *extensions = MM_GCExtensions::getExtensions(env);
+	MM_HeapRegionDescriptorStandard *region = NULL;
+	GC_HeapRegionIteratorStandard regionIterator(extensions->heapRegionManager);
+
+	if (env->_currentTask->synchronizeGCThreadsAndReleaseSingleThread(env, UNIQUE_ID)) {
```

Comment:
why this empty single threaded section needed? I guess it is some kind of leftover

Reply:
If we need to sync threads here (?) `MM_ParallelTask::synchronizeGCThreads()` should be used instead.

Reply:
iterateAllContinuationObjects() is called just after scanContinuationObjects() in GC clearable phase, scanContinuationObjects() would rebuild/update the continuation lists, iterateAllContinuationObjects() kind depends on the result of scanContinuationObjects(), both methods are running parallel, it is possible that iterateAllContinuationObjects() pick one continuation sub list has been updated by scanContinuationObjects(), so we need sync threads here, yes, I will update it to MM_ParallelTask::synchronizeGCThreads().